docs/docs/components/featurizers.mdx

```diff
@@ -330,6 +330,10 @@ Note: The `feature-dimension` for sequence and sentence features does not have t
   If you want to use character n-grams, set `analyzer` to `char` or `char_wb`.
   The lower and upper boundaries of the n-grams can be configured via the parameters `min_ngram` and `max_ngram`.
   By default both of them are set to `1`.
+  By default the featurizer takes the lemma of a word instead of the word directly if it is available. The lemma
+  of a word is for example set by the `SpacyTokenizer`. You can disable this behavior
```

Comment:
Sorry if I missed something, but should we mention that we provide implementation only for `SpacyTokenizer`?